### PR TITLE
Raises the stack size of ATK and ABKs to 10, tweaks the Medical Bay's total stock of these to keep them in line

### PR DESCRIPTION
**Pull request by sheeplets**
Doesn't affect regular gauze and ointment, just the advanced variants. Also Space Klot, which isn't in the game currently.

• Increases the stack size of Advanced Trauma Kits and Advanced Burn Kits from 5 to 10.
• NanoMed Plus vendors and advanced first-aid kits have had their stock shuffled to accommodate. The Medical Bay now has a total of 90 ATKs and 60 ABKs. Reduced down from the 90 ATKs and 75 ABKs we have now.
• Advanced first-kits have been given a coagzolug autoinjector and a health analyzer to fill in empty space.
• Brute first-aid kits have been brought down from 2 ATKs to 1 - total amount unchanged. Added an inaprovaline autoinjector into them to fill the space.

### CODE REVIEW

**Comment by sheeplets** [comment]
!review

**Review by SleepyGemmy — APPROVED** [review]

File: html/changelogs/sheeplets.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - balance: "Increased the stack size of Advanced Trauma Kits and Advanced Burn Kits to 10. Up from 5.
```
Comment (SleepyGemmy) [inline html/changelogs/sheeplets.yml:40]:
```suggestion
  - balance: "Increased the stack size of Advanced Trauma Kits and Advanced Burn Kits to 10. Up from 5."
```

Comment (sheeplets) [reply html/changelogs/sheeplets.yml:40]: image

**Review by alsoandanswer — DISMISSED** [review]
Should be discussed for balance

I don't particularly see the need for the number of ATK and ABKs to be effectively doubled.

**Comment by sheeplets** [comment]
> I don't particularly see the need for the number of ATK and ABKs to be effectively doubled.

Fair-o. Give me a second while I fight for my life with GitKraken.

**Comment by sheeplets** [comment]
We now have the same total number of ATKs as before, and slightly _fewer_ ABKs available.

**Comment by SleepyGemmy** [comment]
balance issues addressed.

**Review by SleepyGemmy — APPROVED** [review]

**Comment by sheeplets** [comment]
It'd probably be a good idea to test-merge this for a few rounds, just to see if people fight over supplies.

**Comment by anconfuzedrock** [comment]
having larger stack sizes with fewer stacks available runs the risk of people hogging them without even trying. I don't like this.

**Comment by sheeplets** [comment]
> having larger stack sizes with fewer stacks available runs the risk of people hogging them without even trying 'cause grabbing one stack is now a double dose. I don't like this.

Hence the request for a test merge.

**Comment by sheeplets** [comment]
Nah this is shit

**Comment by EmperorJayJay** [comment]
I know this is closed but I believe medical has 4 advanced brute and advanced fire things in each medvendor. Maybe bringing it to 5 in them? Idk It seems mostly fine as is

**Pull request closed without merging**